docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -693,3 +693,193 @@ To handle the renewal, you can use the periodic reloading mechanism:
 %prod.quarkus.http.insecure-requests=disabled
 ----
 
+== Quarkus CLI commands and development CA (Certificate Authority)
+
+The TLS registry provides CLI commands to generate a development CA and trusted certificates.
+This avoids having to use self-signed certificates locally.
+
+[source, shell]
+----
+> quarkus tls-registry
```

Comment:
The config namespace is `quarkus.tls`. Do we want to be consistent at the CLI level and have `quarkus tls`? It might end up being a source of conflict though so I would understand if you thought it wouldn't be a good idea.

Reply:
Unfortunately, it is not possible to configure the "tls-registry" part when implementing a CLI plugin. It's the extension name that is reused (and it does not seem to be configurable). We can rename the extension, but that would be a massive change.

Reply:
Well, if you know a way, I'm very interested! 

@iocanel maybe?

Reply:
I recently pushed a change that is supposed to allow for aliasing. I will comment below with how its meant to be done.

Reply:
I set the alias to `tls`, so much better!